variables.tf

```diff
@@ -172,39 +172,39 @@ variable "container_number_map" {
       frontend = 1
       nvme     = 1
       nics     = 4
-      memory   = "31GB"
+      memory   = ["33GB","31GB"]
```

Comment:
what is this?

Reply:
new memory compute according to num of compute containers

Reply:
so this isn't relevant to frontend PR